app/src/main/res/layout/activity_main.xml

```diff
@@ -17,4 +17,15 @@
         android:layout_width="match_parent"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
+
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
+        android:id="@+id/fab"
+        style="@style/FloatingButtons"
+        android:layout_width="@dimen/fab_circumference_size"
+        android:layout_height="@dimen/fab_circumference_size"
+        android:src="@android:drawable/ic_input_add"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/recyclerView" />
```

Comment:
```suggestion
        app:layout_constraintTop_toBottomOf="@id/recyclerView" />
```
remove the "+" symbol from the constraint

Reply:
Done thx